fuel-core/src/lib.rs

```diff
@@ -1,4 +1,3 @@
-pub mod chain_config;
 pub mod database;
```

Comment:
maybe we should replace this with the following to avoid breaking imports downstream?
```suggestion
pub use fuel_chain_config as chain_config;
pub mod database;
```